languages/clojure/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+A `string` in Clojure is a Java string, which is an object that represents immutable text as a sequence of Unicode characters (letters, digits, punctuation, etc.) and is defined as follows:
```

Comment:
Do students need to know that strings are actual Java strings to solve the exercise? If not, this mention should be removed from this document (it is perfect for an after.md document).